.storybook/addons.js

```diff
@@ -1,4 +1,45 @@
 import '@storybook/addon-actions/register'
 import '@storybook/addon-links/register'
 import '@storybook/addon-a11y/register'
-import '@storybook/addon-viewport/register'
+import registerScissors from 'storybook-addon-scissors'
+
+const devices = [
```

Comment:
I created this list of devices with widths inside the 'range' of each of our viewports (all different mobile, tablet sizes plus desktop)

Reply:
Nice!